java-action-archetype/README.md

```diff
@@ -0,0 +1,14 @@
+# Maven Archetype for Java Action
+
+This archetype helps to generate the JavaAction template project
```

Comment:
Can we also include the dependencies, as if the developer wouldn't have a java environment setup ?

Reply:
am not getting you, mean what you mean by dependencies ?

Reply:
I.e. `Install Java 9 or 8` or `Install maven version ...` ?

Reply:
There is no deps extra needed @ddragosd - Java8 is minimum for JavaAction in OW so thats default and maven version later the better but the archetype does not tie to any maven version.

Reply:
can you try to go through the steps documented here on i.e. `docker run -ti --entrypoint bash openwhisk/invoker`  and see what's missing besides java 8 ?

Reply:
maven required on the Host system where the developer or user will be developing it, added some pre-req to the README